app/src/main/java/org/schabi/newpipe/fragments/list/search/filter/SearchFilterOptionMenuAlikeDialogFragment.java

```diff
@@ -0,0 +1,88 @@
+// Created by evermind-zz 2022, licensed GNU GPL version 3 or later
+
+package org.schabi.newpipe.fragments.list.search.filter;
+
+import android.app.Dialog;
+import android.os.Bundle;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.Window;
+import android.view.WindowManager;
+
+import org.schabi.newpipe.databinding.SearchFilterOptionMenuAlikeDialogFragmentBinding;
+import org.schabi.newpipe.extractor.StreamingService;
+
+import java.util.ArrayList;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.widget.Toolbar;
+import androidx.fragment.app.DialogFragment;
+
+public class SearchFilterOptionMenuAlikeDialogFragment extends BaseSearchFilterDialogFragment {
+
+    private SearchFilterOptionMenuAlikeDialogFragmentBinding binding;
+
+    public static DialogFragment newInstance(
+            final int serviceId,
+            final ArrayList<Integer> userSelectedContentFilter,
+            final ArrayList<Integer> userSelectedSortFilter) {
```

Comment:
See the earlier suggestion.

```suggestion
    public static DialogFragment newInstance(
            final int serviceId,
            final List<Integer> userSelectedContentFilter,
            final List<Integer> userSelectedSortFilter) {
```